docs/paper/paper.md

```diff
@@ -0,0 +1,72 @@
+  - hydrology-stormwater-analysis
+  - swmm5
+  - swmm
+  - hydraulic-modelling
+authors:
+  - name: "Barnaby Dobson"
+    orcid: "https://orcid.org/0000-0002-0149-4124"
+    affiliation: 1
+  - name: "Diego Alonso-Álvarez"
+    orcid: "https://orcid.org/0000-0002-0060-9495"
+    affiliation: 1
+  - name: "Taher Chegini"
+    orcid: "https://orcid.org/0000-0002-5430-6000"
+    affiliation: 2
+affiliations:
+ - name: Imperial College London, UK
+   index: 1
+ - name: Purdue University, US
+   index: 2
+date: 16 October 2024
+bibliography: paper.bib
+---
+
+## Summary
+
+Urban drainage network models (UDMs) are useful for a variety of purposes, most notably simulating and alleviating pluvial flooding. Despite the clear usefulness of UDMs, they are often not used owing to the difficulty and expense of creating them. This creates a significant gap for users attempting to generate UDMs if they are not able to perform an expensive underground survey. A variety of literature exists to derive such UDMs from GIS data, producing hydraulically feasible models that closely approximate real-world systems [@Blumensaat2012-hd;@Chahinian2019-lg;@Reyes-Silva2022-pr]. We identify some key limitations of these approaches, most notably the lack of automatic data acquisition and preprocessing, that all approaches are closed-source to date, and that a key feature of such an approach should be to facilitate extension and customisation.
+
+## Statement of need
+
+SWMManywhere is an open-source Python package designed for the global synthesis of urban drainage networks. SWMManywhere integrates publicly available geospatial data and automates data acquisition and preprocessing, reducing the technical burden on users. The CLI offers a flexible workflow, providing an accessible entry point to using and customising synthesis. Its parameterized design enables detailed sensitivity analyses, allowing users to understand and manage uncertainties inherent in urban drainage modelling [@Dobson2024-dv]. By emphasizing user control, SWMManywhere allows tuning of outputs with parameters to meet local requirements, making it adaptable to a wide range of scenarios. Designed for both researchers and practitioners in urban water management, SWMManywhere responds to the limitations of existing methods by providing an end-to-end, open-source, and customisable solution.
+
+### Data
+
+A variety of datasets were selected to enable SWMManywhere to be applied globally, \autoref{table:table1}.
+
+: SWMManywhere data sources. \label{table:table1}
+
+| Data Source | Description | Reference |
+|-------------|-------------| --------- |
+| **OpenStreetMap (OSM)** | Provides global street and river data, used to define potential pipe locations and outfall points for drainage networks. | [@Boeing2017;@OpenStreetMap] |
+| **Google-Microsoft Open Buildings** | A dataset of global building footprints, used for estimating impervious surfaces essential for runoff calculations. | [@VIDA2023] |
```

Comment:
Since you changed this to Overture, you need to update this item and its reference.